packages/ui/test/forum/components/NewThreadPost.test.tsx

```diff
@@ -48,44 +50,102 @@ describe('UI: Add new post', () => {
     useMyMemberships.active = undefined
   })
 
-  it('Hides editor if no membership is selected', async () => {
-    renderEditor()
-    expect(await screen.findByText('Pick an active membership to post in this thread')).toBeDefined()
-  })
+  describe('In a forum thread', () => {
+    const props: NewPostProps = { type: 'forum', thread: { id: '1', categoryId: '1', title: 'thread' } }
+
+    it('Hides editor if no membership is selected', async () => {
+      renderEditor(props)
+      expect(await screen.findByText('Pick an active membership to post in this thread')).toBeDefined()
+    })
+
+    it('Disables the post button if text is empty', async () => {
+      useMyMemberships.setActive(getMember('alice'))
+      renderEditor(props)
+      expect(await getButton('Post a reply')).toBeDisabled()
+    })
 
-  it('Disables the post button if text is empty', async () => {
-    useMyMemberships.setActive(getMember('alice'))
-    renderEditor()
-    expect(await getButton('Post a reply')).toBeDisabled()
+    it('Opens the modal', async () => {
```

Comment:
```suggestion
    it('...', async () => {
```

Should describe the action or state not what happens